static/js/filter.js

```diff
@@ -216,6 +216,7 @@ function encodeOperand(operand) {
 }
 
 function decodeOperand(encoded, operator) {
+    encoded = encoded.replace(/"/g, '');
```

Comment:
Is this decoding supposed to reverse the encoding?  I haven't read this code carefully, but I'm not sure why it's being translated to the empty string here (and why we're not translating `%22` back somewhere).